config-tests/test_server.py

```diff
@@ -0,0 +1,28 @@
+import configparser
+import pytest
+import requests
+
+@pytest.fixture
+def conf():
+    config = configparser.ConfigParser()
+    config.read('manifest.ini')
+    return config
+
+
+def test_heartbeat_fields(env, conf):
+    response = requests.get(conf.get(env, 'we_server_url') + '/__heartbeat__')
+    data = response.json()
+    fields = {'storage', 'portier', 'cache', 'attachments', 'permission', 'oauth'}
```

Comment:
The list of fields depends on the capability configured for the server.
For instance kinto-webextension will have oauth but not kinto-settings, also at Mozilla no stacks currently uses portier.